s/elements/context/element.ts

```diff
@@ -21,7 +22,9 @@ export const NubContext = element<{
 	}).render(use => {
 
 	const [actions] = use.state(stateForActions)
-	const [bindings] = use.state<Bindings>(use.element.bindingsJson ?? defaultBindings)
+	const [bindings, setBindings] = use.state<Bindings>(loadLocalStorageBindings() ?? use.element.bindingsJson ?? defaultBindings)
+
+	listenForLocalStorageBindings(setBindings)
```

Comment:
this is being called on every render, and it adds a new `storage` event listener on the window, every time.

instead, we only want to add one storage listener.

i think we should do something like this instead:

```js
use.setup(() => {
  const listener = setupStorageEventHandling(setBindings)
  window.addEventListener("storage", listener)
  return () => window.removeEventListener("storage", listener)
})
```

that's a proper setup that will even unsubscribe the listener when the component is disconnected.

it's best if we extract that logic out into another file, so it's just:

```js
use.setup(listenForStorageEventsToUpdateBindings(window, setBindings))
```